pymtl3/passes/sverilog/util/test_utility.py

```diff
@@ -324,14 +322,14 @@ def tv_out( model, test_vector ):
 
 def closed_loop_component_test( dut, data, backend = "sverilog" ):
   """Test the DUT with the given test_vector.
-  
+
   User who wish to use this method should pass in the hypothesis data
   strategy instance as `data`. This method will reflect on the interfaces
   and ports of the given DUT and generate input vector.
   """
   # Method to feed data into the DUT
   def tv_in( model, test_vector ):
-    for name, data in test_vector.iteritems():
+    for name, data in test_vector.items():
       exec( "model." + name + " = data" )
```

Comment:
```python
setattr(model, name, data)
```

Faster, and less likely to make security people twitchy when pymtl is adopted elsewhere.

Reply:
`name` here might be an index into an array attribute (`arr[42]`) in which case `setattr` silently fails to set the correct value.

Reply:
Maybe a comment to that effect then?